Sources/AdminPanel/Providers/AdminPanelProvider.swift

```diff
@@ -132,13 +132,13 @@ public final class AdminPanelProvider<U: AdminPanelUserType>: Provider {
 
         let tags: MutableLeafTagConfig = try container.make()
         tags.use([
-            "adminpanel:avatarurl": AvatarURLTag(),
+            "adminpanel:avatarUrl": AvatarURLTag(),
```

Comment:
I think it's better to be consistent with `URL` capitalization.

Reply:
Agree - updated. Also renamed `adminpanel:...` to `adminPanel:..`